.github/workflows/cut-nightly.yml

```diff
@@ -12,7 +12,7 @@ permissions:
 
 jobs:
   cut-nightly:
-    environment: release_tagger
+    environment: cut_nightly
```

Comment:
@danielrozenberg can you create this env and `AMPPROJECTBOT` secret please?

Reply:
Done! I assume [`repo:public`](https://docs.github.com/en/developers/apps/building-oauth-apps/scopes-for-oauth-apps#available-scopes) is enough?

Reply:
thanks, it should be